rcsbsearchapi/search.py

```diff
@@ -360,49 +360,37 @@ class StructSimilarityQuery(Terminal):
     """Special case of a terminal for structure similarity queries"""
 
     def __init__(self, structure_search_type: StructSimEntryType = "entry_id",
-                 value: Optional[str] = None,
-                 input_structure_type: Optional[StructSimInputType] = "assembly_id",
-                 input_option: str = "1",
+                 entry_id: Optional[str] = None,
+                 file_url: Optional[str] = None,
+                 absolute_file_path: Optional[str] = None,
```

Comment:
can you change this to just `file_path`?

Reply:
sure thing!